scootapi/server/setup.go

```diff
@@ -92,3 +95,21 @@ func RunServer(bag *ice.MagicBag, schema jsonconfig.Schema, config []byte) {
 	}()
 	log.Fatal("Error serving: ", <-errCh)
 }
+
+// GetConfigText finds the right text for a configFlag.
+// If configFlag looks like a filename (of the form foo.bar where foo and bar are just alphanumeric),
+// read it as an asset.
+// Otherwise, assume it's the literal json text.
+func GetConfigText(configFlag string, asset func(string) ([]byte, error)) ([]byte, error) {
+	if matched, _ := regexp.Match(`[[:alnum:]]*\.[[:alnum:]]`, []byte(configFlag)); matched {
```

Comment:
I agree with being explicit in general.

In this case, I'm not concerned because text.text isn't valid JSON.

Maybe this step could *also* go into the server.RunServer (so it calls the function that's now in jsonconfig)?